pymc_marketing/clv/models/basic.py

```diff
@@ -129,8 +130,15 @@ def fit(  # type: ignore
         """
         self.build_model()  # type: ignore
 
+        if fit_method:
+            method = fit_method
+            raise UserWarning(
```

Comment:
This just warn first instead of raise

Reply:
It should also be a `DeprecationWarning` 😬.  Changed it to a warning so it doesn't raised user warnings don't break anyone's code.